examples/pydantic_plugin/pydantic_plugin/pydantic_integration_example.py

```diff
@@ -0,0 +1,50 @@
+# %% [markdown]
+# (pydantic_integration_example)=
+#
+# # Pydantic Integration Example
+#
+# Pydantic is a data validation and settings management library for Python, enabling the creation of data models with type annotations.
+#
+# Flyte leverages Pydantic for robust input validation and serialization, ensuring that task inputs are correctly structured.
+
+# %%
+from typing import List
+
+from flytekit import task, workflow
+from flytekit.types.file import FlyteFile
+from pydantic.v1 import BaseModel
```

Comment:
pydantic v2 should work: https://github.com/flyteorg/flytekit/pull/2217

could you run this workflow on Union serverless/flyte sandbox and share with us the result of the execution?

Reply:
I am currently experiencing difficulties while executing my code on the Union serverless/Flyte sandbox. Specifically, I have encountered errors related to importing the `BaseModel` from the Pydantic library.

1. When attempting to execute the following code:
   ```python
   from pydantic.v1 import BaseModel
   ```
I got this error: 
![Screenshot 2024-10-26 114446](https://github.com/user-attachments/assets/40a2fc39-43b7-467d-a116-48618f07b40a)

2. When I tried this:
   ```bash
   from pydantic import BaseModel
   ```
I got this error:
![Screenshot 2024-10-26 114642](https://github.com/user-attachments/assets/a2cb46bf-b589-4f75-937b-be2f47ec3c6e)

I am not able to understand how to avoid these errors. How to resolve these errors?

Reply:
you need to define an imagespec and add the plugin to packages: https://docs.flyte.org/en/latest/user_guide/customizing_dependencies/imagespec.html#image-spec-example